Tribler/Core/Modules/gigachannel_manager.py

```diff
@@ -43,34 +44,42 @@ def start(self):
         or subscribed channels require updating.
         """
 
+        # TODO: break this into smaller checks/functions
+        # TODO: account for all kinds of troubles: lost/wrong .mdblob/torrent, etc
```

Comment:
Have you noticed question marks on some of these TODOs? That is because it is still not clear if these are important.
I would rather not add issues for optimizing the code that is still not merged.

When we merge it, we can grep the codebase for TODOs and create a separate issue for each one, if it is worth it.

Reply:
> Have you noticed question marks on some of these TODOs? That is because it is still not clear if these are important.

I think we should decide whether they are important/viable or not before merging the PR. We already have many TODOs in the codebase and if we do not properly discuss them or be aware of them before they are merged, they will clutter the code base.